server.js

```diff
@@ -12,12 +14,26 @@ app.set('views', path.join(__dirname, 'views'));
 //view engine setup
 app.set('view engine', 'jade');
 
-//get request
-app.get('/', function(req,res){
-  res.render('index');
+app.post('/index', function(req,res){
+  console.log(req.body);
+  res.json(req.body);
 });
 
-//app listener
+app.get('/index',function(req,res){
+  var title = ["one","two"];
+  res.json(title);
+  console.log(title);
+});
+app.delete('/index',function(req,res){
+  req.body.remove;
+  res.json("That sucker has been destroyed");
+});
+app.put('/index',function(req,res){
+  req.body.find;
+  req.body.remove;
+  req.body.save;
+  res.json("Changed");
```

Comment:
Placeholders. We were brainstorming on how we were going to do the put and delete requests. We figured it out since that time.